docs/reference/commandline/daemon.md

```diff
@@ -213,11 +213,23 @@ options for `zfs` start with `zfs`.
 *  `dm.basesize`
 
     Specifies the size to use when creating the base device, which limits the
-    size of images and containers. The default value is 100G. Note, thin devices
-    are inherently "sparse", so a 100G device which is mostly empty doesn't use
-    100 GB of space on the pool. However, the filesystem will use more space for
+    size of images and containers. The default value is 10G. Note, thin devices
+    are inherently "sparse", so a 10G device which is mostly empty doesn't use
+    10 GB of space on the pool. However, the filesystem will use more space for
     the empty case the larger the device is.
 
+    The base device size can be increased at daemon restart which will allow
+    all future images and containers (based on those new images) to be of the 
+    new base device size.
+
+    Example use: 
+
+	$ docker daemon --storage-opt dm.basesize=50G` 
```

Comment:
Can you change the indentation here to 8 spaces, otherwise it won't render as a code-block